cmd/stork/stork.go

```diff
@@ -476,6 +477,12 @@ func runStork(mgr manager.Manager, ctx context.Context, d volume.Driver, recorde
 		adminNamespace = c.String("migration-admin-namespace")
 	}
 
+	// Setting up the generic cache. Currently it has the map to determine the right plural for a crd kind installed in the cluster.
+	if err := genericcache.CreateGenericCache(); err != nil {
+		log.Fatalf("failed to setup generic cache: %v", err)
+	}
+	log.Infof("generic cache has been intialized")
```

Comment:
nit.. crd/<any other name> has been intialized

Reply:
Kept it as generic cache since we agreed upon the package name.